.travis.yml

```diff
@@ -23,18 +24,29 @@ jobs:
       go: 1.9.x
```

Comment:
Depends... I think we should test every version that passes with the current codebase... if it proactively fails because of e.g. a strings.Builder call we can remove it